### ISSUE TITLE
Not working in HA >= 0.96

### ISSUE THREAD

**Issue by astrandb** [echoromeo/hanobo#7]
The Nobö hub component was working fine in 0.95 but it is not working anymore after upgrade to 0.96.
I have activated logging i debug mode and it gave detailed info in 0.95.
Now these are the only relevant logs I get:
```
2019-08-05 14:56:03 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for nobo_hub which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you do experience issues with Home Assistant.
2019-08-05 14:56:03 INFO (MainThread) [homeassistant.loader] Loaded nobo_hub from custom_components.nobo_hub
2019-08-05 14:56:03 INFO (MainThread) [homeassistant.setup] Setting up climate
```
And no devices are created.
The climate integration has undergone a major rewrite in 0.96 that may explain the problem.

**Comment by astrandb** [comment]
Upgraded to 0.97b2 and got some more useful loggings
```
2019-08-05 15:22:33 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for nobo_hub which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you do experience issues with Home Assistant.
2019-08-05 15:22:33 INFO (MainThread) [homeassistant.loader] Loaded nobo_hub from custom_components.nobo_hub
2019-08-05 15:22:33 ERROR (MainThread) [homeassistant.config] Failed to get platform climate.nobo_hub
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.6/site-packages/homeassistant/config.py", line 795, in async_process_component_config
    platform = p_integration.get_platform(domain)
  File "/srv/homeassistant/lib/python3.6/site-packages/homeassistant/loader.py", line 235, in get_platform
    "{}.{}".format(self.pkg_path, platform_name)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/homeassistant/.homeassistant/custom_components/nobo_hub/climate.py", line 16, in <module>
    from homeassistant.components.climate.const import (SUPPORT_OPERATION_MODE, ATTR_TARGET_TEMP_LOW, ATTR_TARGET_TEMP_HIGH, SUPPORT_TARGET_TEMPERATURE_LOW, SUPPORT_TARGET_TEMPERATURE_HIGH, STATE_ECO)
ImportError: cannot import name 'SUPPORT_OPERATION_MODE'
2019-08-05 15:22:33 INFO (MainThread) [homeassistant.setup] Setting up climate
2019-08-05 15:22:33 INFO (MainThread) [homeassistant.setup] Setup of domain climate took 0.0 seconds.
```

**Comment by ghost** [comment]
Hello, got the same issue. I get these logs.
```
Failed to get platform climate.nobo_hub


Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config.py", line 795, in async_process_component_config
    platform = p_integration.get_platform(domain)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 235, in get_platform
    "{}.{}".format(self.pkg_path, platform_name)
  File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/config/custom_components/nobo_hub/climate.py", line 16, in <module>
    from homeassistant.components.climate.const import (SUPPORT_OPERATION_MODE, ATTR_TARGET_TEMP_LOW, ATTR_TARGET_TEMP_HIGH, SUPPORT_TARGET_TEMPERATURE_LOW, SUPPORT_TARGET_TEMPERATURE_HIGH, STATE_ECO)
ImportError: cannot import name 'SUPPORT_OPERATION_MODE' from 'homeassistant.components.climate.const' (/usr/src/homeassistant/homeassistant/components/climate/const.py)
```

Apparently hass did some changes to the climate integration. Maybe that's what causing the issue? (https://developers.home-assistant.io/blog/2019/07/03/climate-cleanup.html)

**Comment by astrandb** [comment]
Fixed by PR #8
